recipes/pyemd/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "pyemd" %}
+{% set version = "0.4.2" %}
+{% set sha256sum = "6585b27df1efe67ac553b4351abf86302365dbddee5695628e621580c4459de3" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256sum }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+  skip: True  # [win32 or linux32]
+  features:
+    - vc9   # [win and py27]
+    - vc10  # [win and py34]
+    - vc14  # [win and py>34]
+
+# Numpy version supported >=1.9
+requirements:
+  build:
+    - toolchain
+    - python
+    - setuptools
+    - numpy x.x
+
+  run:
+    - python
+    - numpy x.x
+
+test:
+  imports:
+    - pyemd
+
+about:
+  home: http://github.com/wmayner/pyemd
+  license: MIT
```

Comment:
Can you please raise an issue or PR with upstream to add it to the `MANIFEST.in`?